package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Unpack a browser type and version from the useragent string",
   "main": "index.js",
   "scripts": {
```

Comment:
Missing `types` property - TypeScript will need this to be able to pull in the types from the resulting package.